WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderWebView.java

```diff
@@ -78,11 +84,21 @@ public boolean isDestroyed() {
 
     public ReaderWebView(Context context, AttributeSet attrs) {
         super(context, attrs);
+
+        TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.BoundedWidth);
```

Comment:
Nice chance to reduce code repetition actually. With cf6faa4, I've opted to move that loading into `init()` instead of shortcircuiting the constructors. Thanks!